_platforms/en/designers.md

```diff
@@ -36,28 +93,42 @@ The goal is to provide kits - documents, guides, or software - that can save tim
 
-* [Introduction to the UI Kit](https://designers.italia.it/kit/ui-kit/){:target="_blank"}
-* [UI Kit components and patterns](https://invis.io/RJFGS2UC3HS){:target="_blank"}
+* [Introduction to the UI Kit](https://designers.italia.it/design-system/come-iniziare/per-designer/){:target="_blank"}
+* [UI Kit components and patterns](https://www.figma.com/community/file/1105848677422572920/ui-kit-italia-v3-1-0){:target="_blank"}
 * [GitHub Repository](https://github.com/italia/design-ui-kit){:target="_blank"}
 
-Many Public Administration projects are using the UI Kit to define their own interfaces, including models for [sites of the municipalities](https://github.com/italia/design-comuni-prototipi){:target="_blank"} and [sites of schools](https://github.com/italia/design-scuole-prototipi){:target="_blank"}.
+Many Public Administration projects are using the UI Kit to define their own interfaces, including templates for [sites of the municipalities](https://github.com/italia/design-comuni-prototipi){:target="_blank"} and [sites of schools](https://github.com/italia/design-scuole-prototipi){:target="_blank"}.
 
-For questions and discussions about the UI Kit, on [Developers Italia Slack](https://slack.developers.italia.it/){:target="_blank"}, you can refer to the channel [#design-ui](https://developersitalia.slack.com/messages/C9N62GX8E/){:target="_blank"}.
+For questions and discussions about the UI Kit, on [Developers Italia Slack](https://slack.developers.italia.it/){:target="_blank"}, you can refer to the channel [#design-system-ui-kit](https://developersitalia.slack.com/archives/C9N62GX8E) {:target="_blank"}.
 
 ## Web Development Kit
 
-The [Web Development Kits](https://designers.italia.it/kit/web-development-kit/){:target="_blank"} are a set of tools for the front end development of websites and web applications. They are HTML, CSS and Javascript libraries that allow you to create sites that comply with the Design Guidelines and code the provisions of the UI Kit. As of today, the most stable, updated and mature web development kit is Bootstrap Italia, based on the open-source library Bootstrap 4.
+The [Web Development Kits](https://designers.italia.it/kit/web-development-kit/){:target="_blank"} are a set of tools for the front end development of websites and web applications. They are HTML, CSS and Javascript libraries that allow you to create sites that comply with the Design Guidelines and code the provisions of the UI Kit. As of today, the most stable, updated and mature web development kit is Bootstrap Italia, based on the open-source library Bootstrap 4. They all have been included as part of the design system since 2022. 
```

Comment:
```suggestion
The [Web Development Kits](https://designers.italia.it/kit/web-development-kit/){:target="_blank"} are a set of tools for the front end development of websites and web applications. They are HTML, CSS and Javascript libraries that allow you to create sites that comply with the design guidelines and code the provisions of the UI Kit. As of today, the most stable, updated and mature web development kit is Bootstrap Italia, based on the open-source library Bootstrap 4. They all have been included as part of the design system since 2022. 
```